js/viewer/annotation/arrowline.js

```diff
@@ -0,0 +1,44 @@
+function ArrowLine(attrs, subtype) {
+  attrs = attrs ? attrs : {};
+  Base.call(this, attrs);
+
+  // This tag needs to be line as the HTML tag used for arrow line is line itself
+  this._tag = "line";
+  this._subtype = subtype;
+  this._attrs["x1"] = attrs["x1"] || null;
+  this._attrs["y1"] = attrs["y1"] || null;
+  this._attrs["x2"] = attrs["x2"] || null;
+  this._attrs["y2"] = attrs["y2"] || null;
+  this._attrs["marker-end"] = attrs["marker-end"];
+  this._attrs["data-subtype"] = subtype;
+  this._attrs["data-markerid"] = attrs["markerID"]
```

Comment:
Comment the purpose of these three attributes. Why do you need three and what their values would be

Reply:
Added the required comments.